src/audience.ts

```diff
@@ -1,9 +1,11 @@
 export default class Audience {
-    public id: number;
-    public name: string;
+    public audience_id: number;
 
-    constructor(id: number, name: string) {
-        this.id = id;
-        this.name = name;
+    constructor(
```

Comment:
Looks like the constructor needs to be updated to remove the references to expiration_timestamp_ms